components/memix/memix.app.mjs

```diff
@@ -0,0 +1,32 @@
+import { axios } from "@pipedream/platform";
+import constants from "./common/constants.mjs";
+
+export default {
+  type: "app",
+  app: "memix",
+  propDefinitions: {},
+  methods: {
+    async getTemplates({ $ = this }) {
+      return await axios($, {
+        method: "get",
```

Comment:
```suggestion
      return axios($, {
```
[No need for return await](https://eslint.org/docs/latest/rules/no-return-await) and 'get' is the default method, so you can safely omit it

Reply:
cool. i've updated it